src/ralph/virtual/admin.py

```diff
@@ -310,6 +310,21 @@ def get_cloudproject(self, obj):
     get_cloudproject.allow_tags = True
     get_cloudproject._permission_field = 'parent'
 
+    def get_service(self, obj):
+        if self.service_env_id:
+            return '<a href="{}">{}</a>'.format(
+                reverse(
+                    "admin:assets_service_change",
+                    args=(obj.service_env.service.id,)
```

Comment:
done, but it is not necessary, because I have get str(obj.service_env.service) in line 320